src/concrete/ml/sklearn/neighbors.py

```diff
@@ -0,0 +1,117 @@
+"""Implement sklearn linear model."""
+from typing import Any, Dict
+
+import sklearn.linear_model
+
+from .base import SklearnKNeighborsClassifierMixin
+
+
+# pylint: disable=invalid-name,too-many-instance-attributes
+class KNeighborsClassifier(SklearnKNeighborsClassifierMixin):
+    """A k-nearest classifier model with FHE.
+
+    Parameters:
+        n_bits (int, Dict[str, int]): Number of bits to quantize the model. If an int is passed
+            for n_bits, the value will be used for quantizing inputs and weights. If a dict is
+            passed, then it should contain "op_inputs" and "op_weights" as keys with
+            corresponding number of quantization bits so that:
+            - op_inputs : number of bits to quantize the input values
+            - op_weights: number of bits to quantize the learned parameters
```

Comment:
I am not sure it's clear what you are referring to here with `op_weights` and `op_inputs`.

Reply:
It was initially a copay past from the classifiermixin, I forgot to remove it. 
we don't talk about weights in KNN, but about X_fit
Thanks